gateway/src/main/java/io/zeebe/gateway/impl/broker/BrokerClientImpl.java

```diff
@@ -97,8 +103,35 @@ public BrokerClientImpl(final GatewayCfg configuration, final ActorClock actorCl
     transport = transportBuilder.build();
     internalTransport = internalTransportBuilder.build();
 
+    final String[] parts = configuration.getCluster().getContactPoint().split(":");
+    final Integer port = Integer.parseInt(parts[1]);
+
+    atomix =
+        AtomixCluster.builder()
+            .withMemberId("gateway")
+            .withClusterId(configuration.getCluster().getClusterName())
```

Comment:
The configuration should allow to set the internal gateway address/port, otherwise all gateways will bind to the same default atomix port

Reply:
I would propose in the gateway cluster cfg to add a host/port configuration option, which is used for internal communication.